qcodes/instrument_drivers/Minicircuits/USB_SPDT.py

```diff
@@ -66,8 +66,12 @@ def __init__(self, name: str, driver_path: Optional[str] = None,
                 the dll to open properties and check the 'unblock' checkmark
                 in the bottom. Check that your python installation is 64bit."""
             )
-        import mcl_RF_Switch_Controller64
-        self.switch = mcl_RF_Switch_Controller64.USB_RF_SwitchBox()
+        try:
+            import mcl_RF_Switch_Controller64 as mw_driver
+        except:
+            import mcl_RF_Switch_Controller_NET45 as mw_driver
+
+        self.switch = mw_driver.USB_RF_SwitchBox()
```

Comment:
actually, since the name of the import changes depending on the dll, we should probably add an argument to the driver for that name as well? such that if there's yet another new dll with yet a new name to import, that changes to the driver code are not necessarily needed.

but perhaps that's too much

Reply:
I think it is fine to just check for the two names that are in use. Seems unlikely that they are going to make yet another file. Would assume it is pretty rare that .NET binaries needs to be updated.

Reply:
```mcl_RF_Switch_Controller_NET45``` needs to be added to the list of modules that we don't have types for in `pyproject.toml``` next to `mcl_RF_Switch_Controller64`